relay/tutorials/trustlines_tlbc_system.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Running the Relay with a TLBC-node (with docker)
+sidebar_label: Running the Relay with a TLBC-node (with docker)
+---
+
+The [docker/trustlines/tlbc](https://github.com/trustlines-protocol/relay/tree/master/docker/trustlines/tlbc) directory in the relay
+repository contains the files needed to start components of the trustlines protocol (tlbc-node, relay, indexer) via
+docker-compose. Following these setup instructions will give you a:
+- TLBC-Node with OpenEthereum
+- Py-eth-index connected to the TLBC-Node
+- Relay instance connected to the TLBC-Node
+
+:::caution
+The config by default exposes port 5000 to connect to the relay. It is advised to actually
+put a proxy (nginx, caddy, apache or traefik) in front of the relay server.
+:::
+
+## Services
+
+The docker-compose file contains service definitions for the following services:
+
+- db: A service running a postgres server. The postgres files will be
+  stored in the `postgres-data` docker volume.
+
+- tlbc: A service running a modified OpenEthereum node for the trustlines
+  blockchain. The blockchain data will be stored in the
+  `blockchain-data` docker volume.
+
+- index: A service running py-eth-index, which synchronizes certain
+  information from OpenEthereum into the postgres database.
+
+- relay: The relay server itself.
+
+To find the location of the volumes on the system you can run:
+```bash
+docker volume ls
+```
+
+This will list the created volumes. Then you can execute:
+```
+docker volume inspect --format '{{ .Mountpoint }}' VOLUME_NAME
+```
+
+This will give you the path to the volume data.
+
+
+## Setup
+
+We need to do some initial setup and configuration for the system to work. You
+need to provide the `addresses.json` file, which should be put in the directory
+alongside the `docker-compose.yml` file.
+
+:::note
+The `addresses.json` file specifies which currency networks your py-eth-index & relay instance are
+going to index. If you want to run your relay only with a set of particular currency networks
+adapt the file accordingly.
+:::
+
+If you want the Trustlines Blockchain with the official set of currency networks, you can run:
+
+```bash
+cp ../../../config/addresses_tlbc.json ./addresses.json
+```
+
+:::note
+The `addresses.json` file also includes references to the identity related contracts.
+:::
+
+
+### Fetch Docker Image
+
+Let's first build and fetch all the images that we will need
+without starting any services with the following command:
+
+```bash
+docker-compose up --no-start
```

Comment:
I don't think that is correct. This may create container without starting them but still instantiating them. I believe this could be a problem because if you adapt the node config later, it could be that it is not loaded as the container was already created.

I cannot test it locally because my setup is broken as.

Reply:
You should do `docker-compose pull` if you want to pull. but that shouldn't be necessary.

Reply:
Ok. We did it without that and it worked.